app/helpers/navigation_bar_helper.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module NavigationBarHelper
+  def render_navigation_bar?(provider)
+    request.path.exclude?("support") && provider && !current_page?(root_path) &&
+      !current_page?(publish_provider_path(provider.provider_code)) &&
+      provider.recruitment_cycle
```

Comment:
`Provider` in our app doesn't have `recruitment_cycle`. So I'm assuming this should break.